src/libraries/System.Net.Security/src/System/Net/Security/NetEventSource.Security.cs

```diff
@@ -303,52 +289,52 @@ public void UsingCachedCredential(SecureChannel secureChannel)
         {
             if (IsEnabled())
             {
-                RemoteCertificateError(GetHashCode(secureChannel), message);
+                RemoteCertificateError(GetHashCode(SslStream), message);
             }
         }
         [Event(RemoteCertificateErrorId, Keywords = Keywords.Default, Level = EventLevel.Verbose)]
-        private void RemoteCertificateError(int secureChannelHash, string message) =>
-            WriteEvent(RemoteCertificateErrorId, secureChannelHash, message);
+        private void RemoteCertificateError(int SslStreamHash, string message) =>
+            WriteEvent(RemoteCertificateErrorId, SslStreamHash, message);
 
         [NonEvent]
-        public void RemoteCertDeclaredValid(SecureChannel secureChannel)
+        public void RemoteCertDeclaredValid(SslStream SslStream)
         {
             if (IsEnabled())
             {
-                RemoteCertDeclaredValid(GetHashCode(secureChannel));
+                RemoteCertDeclaredValid(GetHashCode(SslStream));
             }
         }
         [Event(RemoteVertificateValidId, Keywords = Keywords.Default, Level = EventLevel.Verbose)]
-        private void RemoteCertDeclaredValid(int secureChannelHash) =>
-            WriteEvent(RemoteVertificateValidId, secureChannelHash);
+        private void RemoteCertDeclaredValid(int SslStreamHash) =>
+            WriteEvent(RemoteVertificateValidId, SslStreamHash);
 
         [NonEvent]
-        public void RemoteCertHasNoErrors(SecureChannel secureChannel)
+        public void RemoteCertHasNoErrors(SslStream SslStream)
         {
             if (IsEnabled())
             {
-                RemoteCertHasNoErrors(GetHashCode(secureChannel));
+                RemoteCertHasNoErrors(GetHashCode(SslStream));
             }
         }
         [Event(RemoteCertificateSuccesId, Keywords = Keywords.Default, Level = EventLevel.Verbose)]
-        private void RemoteCertHasNoErrors(int secureChannelHash) =>
-            WriteEvent(RemoteCertificateSuccesId, secureChannelHash);
+        private void RemoteCertHasNoErrors(int SslStreamHash) =>
+            WriteEvent(RemoteCertificateSuccesId, SslStreamHash);
 
         [NonEvent]
-        public void RemoteCertUserDeclaredInvalid(SecureChannel secureChannel)
+        public void RemoteCertUserDeclaredInvalid(SslStream SslStream)
         {
             if (IsEnabled())
             {
-                RemoteCertUserDeclaredInvalid(GetHashCode(secureChannel));
+                RemoteCertUserDeclaredInvalid(GetHashCode(SslStream));
             }
         }
         [Event(RemoteCertificateInvalidId, Keywords = Keywords.Default, Level = EventLevel.Verbose)]
-        private void RemoteCertUserDeclaredInvalid(int secureChannelHash) =>
-            WriteEvent(RemoteCertificateInvalidId, secureChannelHash);
+        private void RemoteCertUserDeclaredInvalid(int SslStreamHash) =>
+            WriteEvent(RemoteCertificateInvalidId, SslStreamHash);
```

Comment:
FWIW, the changes in this file are ones someone could theoretically be broken by.  However, we consider this to be a private logging mechanism that's unsupported, so it should be fine.

Reply:
agreed. What we write at the end is just the hash - there is no actual type but somebody can register own listener. 
I at least tried to preserve existing event ID. We _could_ simply retire whole set and create new events. But that would probably not help either if anybody already expects particular format.